docs/en/tutorials/how_to.md

```diff
@@ -0,0 +1,64 @@
+# Tutorial 11: How to xxx
+
+This tutorial collects answers to any `How to xxx with MMDetection`. Feel free to update this doc if you meet new questions about `How to` and find the answers!
+
+## Backbone network using MMClassification
+
+The Hierarchy Registry function proposed in MMCV allows modules in various algorithm libraries of OpenMMLab to call each other. Users can use backbone networks from MMClassification in MMDetection without implementing a network that already exists in MMClassification.
+
+Suppose you want to use `MobileNetV3-small` as the backbone network of `RetinaNet`, the sample code is as the following.
+
+```python
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# please install mmcls>=0.20.0
+# import mmcls.models to trigger register_module in mmcls
+custom_imports = dict(imports=['mmcls.models'], allow_failed_imports=False)
+pretrained = 'https://download.openmmlab.com/mmclassification/v0/mobilenet_v3/convert/mobilenet_v3_small-8427ecf0.pth'
+model = dict(
+    backbone=dict(
+        _delete_=True, # Delete the backbone field in _base_
+        type='mmcls.MobileNetV3', # Using MobileNetV3 from mmcls
+        arch='small',
+        out_indices=(3, 8, 11), # Modify out_indices
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint=pretrained,
+            prefix='backbone.')), # Need to add prefix to ensure that pre-training weights can be loaded normally
```

Comment:
Should indicate why.
```
# The pre-trained weights of backbone network in MMCls has prefix='backbone.'. The prefix in the keys will be removed so that these weights can be normally loaded.
```